### DISCUSSION TITLE
Allow field Instructions to be customized per-channel they are shared by

### DISCUSSION THREAD

**Discussion by pyrobob83** [ExpressionEngine/ExpressionEngine#2951] [Feature Requests]
Now that individual fields can be shared by multiple channels, it's often the case that the instructions for the field need to differ between channels.  I'll let you guys think up the best way to implement this, but my suggestion would be leaving the default Instructions as part of the field, but allowing an override when setting up Publish Layouts.  This would be non-breaking for past site upgrades but still allow for this feature.

It's worth considering other field settings this could potentially extend to as well... for example, the 'Required field' toggle could also be forced at Field level or at Publish Layout level allowing the same field to be required in one channel, but not another.

**Comment by intoeetive** [comment]
Not exactly what is being requested here, but we plan to add the ability to modify the field label in the layouts in this PR https://github.com/ExpressionEngine/ExpressionEngine/pull/3881

Perhaps we could do something similar with instructions, but also a different approach might be needed
